src/imgtools/cli/autopipeline.py

```
@@ -0,0 +1,267 @@
+        roi_ignore_case=roi_ignore_case,
+        roi_handling_strategy=ROIMatchStrategy[roi_strategy],
+        roi_allow_multi_key_matches=roi_allow_multi_matches,
+        roi_on_missing_regex=ROIMatchFailurePolicy[roi_on_missing_regex],
+        spacing=spacing,
+        window=window,
+        level=level,
+    )
+    
+    # Run the pipeline
+    try:
+        results = pipeline.run()
```

Ruff (0.8.2)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)